editor/src/uuiui/icn.tsx

```diff
@@ -3,8 +3,22 @@ import * as React from 'react'
 import { betterReactMemo } from '../utils/react-performance'
 import { getPossiblyHashedURL } from '../utils/hashed-assets'
 import { Tooltip } from './tooltip'
+import { useColorTheme } from '.'
```

Comment:
```suggestion
```

Very minor, but it looks like this import wasn't used in the end

Reply:
also, what the hell is `import from '.'`

Reply:
Let's just get this line removed from the code and then we can pretend it never happened and never speak of it again